db/warehouse_structure.sql

```diff
@@ -16847,34 +16849,6 @@ CREATE TABLE public.recent_report_enrollments (
 );
 
 
---
```

Comment:
I don't know why there are so many changes in the db schema from migrations unrelated to this PR. Should I go through and manually remove these?

Reply:
This is fine, there's some difference between each developer's DB setup, not sure exactly why, but this file gets generated from a SQL dump instead of a sum of the changes from migrations, so at times it gets thrashed a bit.  As long as the tests continue to pass, it should be close-enough to correct.